src/transforms/remap.rs

```diff
@@ -809,6 +733,49 @@ mod tests {
         assert!(tform.runner().runtime.is_empty());
     }
 
+    #[test]
+    fn remap_return_raw_string_vector_namespace() {
+        let initial_definition = Definition::default_for_namespace(&[LogNamespace::Vector].into());
+
+        let event = {
+            let mut metadata = EventMetadata::default()
+                .with_schema_definition(&Arc::new(initial_definition.clone()));
+            // the Vector metadata field is required for an event to correctly detect the namespace at runtime
+            metadata
+                .value_mut()
+                .insert(&owned_value_path!("vector"), BTreeMap::new());
```

Comment:
Strikes me as something we may want to add as a helper as we need to test more Vector namespaced events.

Reply:
Definitely. I generally like seeing a couple more examples of how it's used then I'll probably convert it to some kind of helper.